gradebookng/impl/src/main/java/org/sakaiproject/grading/impl/GradingServiceImpl.java

```diff
@@ -4817,6 +4818,12 @@ public Gradebook getGradebook(String uid) {
         return gradingPersistenceManager.getGradebook(uid).orElseGet(() -> addGradebook(uid));
     }
 
+    @Transactional
+    public void initGradebook(String uid) {
+
+        getGradebook(uid);
```

Comment:
should it do a site lookup before simply creating a gradebook with some string?

Reply:
Ok, it will be done.